### ISSUE TITLE
EU Privacy Information

### ISSUE THREAD

**Issue by tobiasschmidt89** [denoland/deploy_feedback#272]
I would like to host a website on deno deploy in Germany. I did not find official statement or guide if deno deploy is in check with GDPR, Schrems 2, DPA, etc. It would be great if you could provide information, features and or processes on this topic so that it is "safe" for users in EU to host websites and apps on deno deploy.

e.g. I found [this post](https://segment.com/blog/regional-segment/) by software provider Segment.com that has some information on how they try to solve GDPR, Schrems 2 (e.g. they seem to have a sub company and infrastructure in EU/non-EU to split processing). So something like this would be super helpful for me to feel confident in using deno deploy in Germany.

Thanks

**Comment by ije** [comment]
thanks, we will fix!

**Comment by felix-schindler** [comment]
It would be very helpful to be able to select which region to host instead of "earth". Before that, it is impossible for many companies to use the service.

**Comment by cetinsert** [comment]
It would be better if we could just
- [ ] https://github.com/denoland/deploy_feedback/issues/127
This would enable more scenarios than just conforming to the rules of 1 region which should not be held above other regions in terms of developer resources dedicated to satisfying its whimsical needs.

**Comment by tobiasschmidt89** [comment]
> It would be better if we could just
> - [ ] https://github.com/denoland/deploy_feedback/issues/127
> This would enable more scenarios than just conforming to the rules of 1 region which should not be held above others in terms of developer resources dedicated to satisfying its whimsical needs.

Makes sense to have this option, but I am not sure if it is sufficient to be in check with GDPR. 

E.g. it might be necessary to have a deno deploy sub company in EU to be in check with Schrems 2. As far as I understand the US cloud act, it affects also data processed/collected in other countries by US companies. Therefore to keep the EU citizens data isolated would require a physical and "legal" isolation. 

Additional I think it would be necessary to have a way to create a DPA with deno deploy and a post explaining the data processing (What Data, Where it is processed, How it is processed, Who has access, ...) so that it is easy to explain in a Privacy Policy and to have a reference in case of an issue.

PS: Again I am no expert on this topic, so everything I explain here might be wrong or insufficient to be in check with GDPR, etc.

**Comment by eqoram** [comment]
Also very interesting for our organization! Any updates on this?

**Comment by mknet** [comment]
Hello everbody,

it would be a huge step to solve this issue for all people try to use Deno Deploy within Europe / the EU.

Any progress regrading GDPR?

Thanks and regards
Marcel

**Comment by CetinSert** [comment]
You know you can all just deploy deno on your traditional VMs, right!?

They have only 3 regions in EU anyway:
![image](https://github.com/user-attachments/assets/25a3f0c3-d28b-45fb-8f24-ad79388b4c84)

**Comment by Huberst** [comment]
Hi everyone, any progress on this topic? I would really like to use this service, but I can't risk to get sued.
